Templates/Android/generated/BaseEntityCollectionReferenceRequest.java.tt

```diff
@@ -39,37 +39,43 @@
                 .post(new<#=TypeName(c)#>, body);
     }
 
+<# if (c.GetFeatures().CanExpand) { #>
     /**
      * Sets the expand clause for the request
      *
      * @param value The expand clause
      * @return The updated request
      */
     public <#=ITypeCollectionReferenceRequest(c)#> expand(final String value) {
-        addQueryOption(new QueryOption("expand", value));
+        addQueryOption(new QueryOption("$expand", value));
         return (<#=TypeCollectionReferenceRequest(c)#>)this;
     }
 
+<# } #>
+<# if (c.GetFeatures().CanSelect) { #>
```

Comment:
If this means more string builder code, then I'd say don't make that change. We are trying to slowly go away from that model since it is not maintainable or manageable in a reasonable way. I'd much prefer pure templates.